src/components/GroceryItem/GroceryItem.js

```diff
@@ -17,6 +30,7 @@ const GroceryItem = ({ item }) => {
     return groceryItem
       .update({
         purchaseDate: Date.now(),
+        nextPurchase: workOutNextPurchDate(item.howSoon),
```

Comment:
To use the calculateEstimate method from the lib directory, I believe we'll need data for the parameters lastEstimate (which is nextPurchase - purchaseDate), latestsInterval (which is Date.now - purchaseDate), and numberOfPurchases, so we might also want to add a numberOfPurchases field to the database when we initially add an item, setting that field to zero. Then when we update an item because it has been marked as purchases, we'll also need to add 1 to that field.